GitHub-Test-DHprojects/tyler-gittest4.md

```diff
@@ -0,0 +1,15 @@
+# Git Test 4
+
+## Blues Analysis Project
+
+The Kumu networks used to break down different decades are really effective for the viewers' pleasure of seeing information that has been gathered by the two students on this project. It was nice seeing that they were able to gather their lyrics after the unfortunate "link rot" dilemma that had halted their research. I appreciate the breakdown of their research questions on the opening page of the project. It lets the reader know what to expect from the information and gives the page a purpose. 
+
+ 
+
+Ideas that I can gather from this project is something to do with video game companies and games that they have released over the years. Seeing how those games perform on certain platforms and which platforms they are released to. It could show a lot of popularity fluctuations based on time period and platform release. 
+
+## Profanities in Quentin Tarantino's Screenplays
+
+The way that this project was constructed was well thought out. The visuals were not too much nor were they too little. They gave the viewers of this website a good idea of what was being researched even if one was not to read all the information in the different tabs. The visualization that stood out the most to me personally was the weirdly distorted line chart. It popped off the page into my eyes and intrigued me to learn what it was all about. I would like to know why they chose that sort of chart to express their research findings. The chart that I am referring to shows the number of times profanity words is used by either male or female within Quentin Tarantino's screenplays. For a new project, this gives me ideas to break down lyrics in an artist's discography to see what words or phrases they tend to lean towards to bridge gaps in their songs. 
```

Comment:
I agree here with the visuals being just the right amount. They are enough to display the information effectively and they importantly remain inobtrusive.